resources/OptionGroup_resource_demand_types.mgd.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+return [
+  [
+    'name' => 'OptionGroup_resource_demand_types',
+    'entity' => 'OptionGroup',
+    'cleanup' => 'unused',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'resource_demand_types',
+        'title' => 'Resource Demand Types',
+        'description' => NULL,
+        'data_type' => NULL,
+        'is_reserved' => TRUE,
+        'is_active' => TRUE,
+        'is_locked' => FALSE,
+        'option_value_fields' => [
+          'name',
+          'label',
+          'description',
+        ],
+      ],
```

Comment:
Safeguard to prevent errors if the group already exists:
```suggestion
      ],
      'match' => ['name'],
```